test/test_supermodular.jl

```diff
@@ -0,0 +1,26 @@
+using Games
+
+if VERSION >= v"0.5-"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
```

Comment:
you shouldn't be doing this here. You should remove everything above this line and just include this file in `runtests.jl`.